packages/salesforcedx-vscode-lwc/test/vscode-integration/testSupport/testRunner/taskService.test.ts

```diff
@@ -13,9 +13,22 @@ import {
 } from '../../../../src/testSupport/testRunner/taskService';
 
 describe('Task Service Unit Tests', () => {
-  let executeTaskStub: SinonStub;
-  let onDidStartTaskStub: SinonStub;
-  let onDidEndTaskStub: SinonStub;
+  let executeTaskStub: SinonStub<
```

Comment:
these types look a bit painful to maintain. wondering if we can simplify this somehow, especially if the editor isn't going to tell you when they are wrong

Reply:
So when it reports the error, it does tell you what the value should become. I just couldn't get the quick fix to address it.